meetings/2018-01-31.md

```diff
@@ -0,0 +1,127 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-01-31
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=a6YtLIaqMqs 
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/478
+* **Minutes Google Doc**: https://docs.google.com/document/d/1R7Z9Rt01fqFddMnllJddezKwGIbaCcrkrGfw5Lhc_1A
+
+## Present
+
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* ......... ...... ......... @ChALkeR (TSC)
```

Comment:
Hmm

Reply:
I thought I had fixed that, but seems not to have worked.

Reply:
Looks ok if I use "view".

Reply:
So I think its ok.

Reply:
I fixed it after Timothy's comment.